packages/react/__tests__/__snapshots__/exports.ts.snap

```diff
@@ -111,6 +111,7 @@ exports[`@aws-amplify/ui-react/internal exports should match snapshot 1`] = `
   "IconVisibilityOff",
   "IconWarning",
   "PrimitiveCatalog",
+  "STORAGE_BROWSER_ICON_PATHS",
```

Comment:
Should not be a public export, needs to be exported from _packages/react/src/internal.ts_